scripts/ci/ci.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+set -xe
+
+source scripts/set_nixpath.sh
+
+OS_NAME=$(uname -s | tr A-Z a-z)
+
+if [[ ("$OS_NAME" == "linux") && ("$BUILDKITE_BRANCH" == "master") ]];
+  then with_haddock=true
+  else with_haddock=false
+fi
+
+targets="cardano-sl cardano-sl-auxx cardano-sl-tools cardano-sl-wallet"
+
+# There are no macOS explorer devs atm and it's only deployed on linux
+if [[ "$OS_NAME" == "linux" ]]; then
+   targets="$targets cardano-sl-explorer-static"
+fi
+
+# TODO: CSL-1133: Add test coverage to CI. To be reenabled when build times
+# become smaller and allow coverage report to be built.
+#projects="core db lrc infra update ssc godtossing txp"
+#to_build=''
+
+#for prj in $projects; do
+#  to_build="$to_build cardano-sl-$prj"
+#done
+
+for trgt in $targets; do
+  echo "Prebuilding dependencies for $trgt, quietly.."
+  nix-shell -A $trgt --run true --no-build-output --cores 0 --max-jobs 4 default.nix ||
```

Comment:
This rebuilds targets now twice, we should only build verbosely.

Also we should add `cardano-sl-auxx` to release.nix to prebuilt deps.

Reply:
Ok, will do.  (The second rebuild only happens for a failing dependency, though.)